docs/conf.py

```diff
@@ -189,11 +197,59 @@
 from gammapy.utils.docs import gammapy_sphinx_ext_activate
 gammapy_sphinx_ext_activate()
 
-# copy notebooks
-if os.environ['GAMMAPY_EXTRA']:
+# ---
+# notebooks
+# make relative links
+DOWNLOAD_CELL = """
+<div class='admonition note'>
+[Download notebook](../_static/notebooks/nbfilename.ipynb)
+
+This is a *fixed-text* formatted version of a Jupyter notebook. """
+if on_rtd:
+    DOWNLOAD_CELL += """
+    You may download the whole HTML documentation for this version of gammapy
+    using the link at the bottom of this page and execute the notebooks
+    in your local desktop inside the `_static/notebooks/` folder.
+    """
+DOWNLOAD_CELL += """</div>"""
+
+url_docs = setup_cfg.get('url_docs')
+
+def modif_links(folder):
+    for filename in os.listdir(folder):
+        filepath = os.path.join(folder, filename)
+        if os.path.isfile(filepath) and filepath[-6:] == '.ipynb':
+            if folder=='notebooks':
+                strcell = DOWNLOAD_CELL.replace('nbfilename.ipynb', filename)
+                nb = nbformat.read(filepath, as_version=nbformat.NO_CONVERT)
+                nb.cells.insert(0, new_markdown_cell(strcell))
+                nbformat.write(nb, filepath)
+            with open(filepath, "r") as f:
+                txt = f.read()
+            if folder=='notebooks':
+                txt = re.sub(url_docs+'(.*?)html(\)|#)',r'..\1rst\2', txt, flags=re.M|re.I)
+            if folder=='_static/notebooks':
+                txt = re.sub(url_docs+'(.*?)html(\)|#)',r'..\/..\1html\2', txt, flags=re.M|re.I)
+            with open(filepath, "w") as f:
+                f.write(txt)
+
+# remove existing notebooks if rebuilding
+if eval(setup_cfg.get('rebuild_notebooks')):
```

Comment:
I find this name "rebuild_notebooks" non-optimal, because it doesn't rebuild them, it's removes the old ones.

Note that we already have this option:
```
python setup.py build_docs --help

  --clean-docs (-l)            Completely clean previous builds, including
                               automodapi-generated files before building new
                               ones
```
It's probably not easy to add to this option or add a new one. But then for now I'd suggest to rename your option in `setup.cfg` to `clean_notebooks`.